platform-sdk/platform-apps/tests/StressTestingTool/src/main/java/com/swirlds/demo/stress/StressTestingToolState.java

```diff
@@ -102,7 +102,10 @@ public void handleConsensusRound(final Round round, final PlatformState platform
     }
 
     private void handleTransaction(@NonNull final ConsensusTransaction trans) {
-        runningSum += ByteUtils.byteArrayToLong(trans.getContents(), 0);
+        if (trans.isSystem()) {
+            return;
+        }
+        runningSum += ByteUtils.byteArrayToLong(trans.getApplicationPayload().toByteArray(), 0);
```

Comment:
Minor: note that `toByteArray()` creates a new primitive bytes array. While it's totally fine in demo code, if you find a need to use a similar call in production code, then it might be worth implementing `Bytes.toLong()` to reduce the load on the GC.

Reply:
yeah, I know. this PR is just for demo apps, didn't want to spend too much time on this since these apps don't even have unit tests. for production, we will not copy